run.py

```diff
@@ -392,6 +392,10 @@ def setup(self):
     def check_skip(self):
         if not "skip" in self.config:
             return
+        if isinstance(self.config["skip"], bool):
+            if self.config["skip"]:
```

Comment:
Nit: Maybe this if check can be skipped since we're checking in the line above the instance type for this var then it must exist.

Reply:
I'm not so sure.. With this change we have to support 3 datatypes for skip.  It could be None, a boolean or a list.

Reply:
Yes, you're right. Sorry about this.